code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -17,22 +16,23 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "across" = 'sound/vox_fem/across.ogg',
 "activate" = 'sound/vox_fem/activate.ogg',
 "activated" = 'sound/vox_fem/activated.ogg',
+"active" = 'sound/vox_fem/active.ogg',
 "activity" = 'sound/vox_fem/activity.ogg',
-"adios" = 'sound/vox_fem/adios.ogg',
 "administration" = 'sound/vox_fem/administration.ogg',
 "advanced" = 'sound/vox_fem/advanced.ogg',
 "aft" = 'sound/vox_fem/aft.ogg',
 "after" = 'sound/vox_fem/after.ogg',
 "agent" = 'sound/vox_fem/agent.ogg',
 "ai" = 'sound/vox_fem/ai.ogg',
+"airlock" = 'sound/vox_fem/airlock.ogg',
 "alarm" = 'sound/vox_fem/alarm.ogg',
+"alcohol" = 'sound/vox_fem/alcohol.ogg',
 "alert" = 'sound/vox_fem/alert.ogg',
 "alien" = 'sound/vox_fem/alien.ogg',
 "aligned" = 'sound/vox_fem/aligned.ogg',
 "all" = 'sound/vox_fem/all.ogg',
 "alpha" = 'sound/vox_fem/alpha.ogg',
 "am" = 'sound/vox_fem/am.ogg',
-"amigo" = 'sound/vox_fem/amigo.ogg',

```

Comment:
Why

Reply:
Unnecessary, especially when we tout "use English here"